dev/KaairaGupta/main.py

```diff
@@ -0,0 +1,38 @@
+from data_loader import load_split_preprocessed_data
+from Ml_functions import model
+from sklearn.ensemble import RandomForestClassifier # Random forest classifier
+from sklearn.tree import DecisionTreeClassifier # for Decision Tree classifier
+from sklearn.svm import SVC # for SVM classification
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import train_test_split # to split the data
+from sklearn.model_selection import KFold # For cross vbalidation
+from sklearn.model_selection import GridSearchCV # for tunnig hyper parameter it will use all combination of given parameters
+from sklearn.model_selection import RandomizedSearchCV # same for tunning hyper parameter but will use random combinations of parameters
+
+#take normal transaction in  0.5 , 0.66 and 0.75  proportion of total data
```

Comment:
please extend the verbosity of this comment, I am not sure what a `transaction` refers to hear.

Reply:
Yes. I have made the comment more intuitive now